src/bin/pgcopydb/cli_stream.c

```diff
@@ -609,7 +609,7 @@ cli_stream_cleanup(int argc, char **argv)
 	bool service = false;
 
 	if (!copydb_init_workdir(&copySpecs,
-							 NULL,
+							 streamDBoptions.dir,
```

Comment:
We can delete this commit https://github.com/dimitri/pgcopydb/pull/497/commits/ce02dc435e199d4ed7975cbcfbfc0f9312ba1678 from this PR.

Reply:
I believe that because I'm going “squash & merge” anyway, that won't be visible once merged, in a way that it's not important to clean-up the history of the PR after review. It might still be useful at times during review, of course, this PR does not look complex enough (yet?) to warrant history cleanup in my eyes.